sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md

```diff
@@ -383,6 +385,9 @@ When filing GitHub issues, the following details are requested for all scenarios
   - 45 (EventProcessorClaimOwnershipError)
   - 103 (EventProcessorLoadBalancingCycleSlowWarning)
   - 104 (EventProcessorHighPartitionOwnershipWarning)
+  - 123 (EventProcessorProcessingHandlerStart)
```

Comment:
Just checking that this one is intentionally omitted from the new events in the above change.

Reply:
I totally messed up the numbering in the other section...